wazo_chatd/plugins/presences/plugin.py

```diff
@@ -38,12 +38,13 @@ def load(self, dependencies):
         initiator = Initiator(dao, auth, amid, confd)
         status_aggregator.add_provider(initiator.provide_status)
 
+        initiator_thread = InitiatorThread(initiator)
+
         if initialization['enabled']:
             thread_manager = dependencies['thread_manager']
-            initiator_thread = InitiatorThread(initiator)
             thread_manager.manage(initiator_thread)
 
-        bus_event_handler = BusEventHandler(dao, notifier)
+        bus_event_handler = BusEventHandler(dao, notifier, initiator_thread)
```

Comment:
Be careful, we want to keep a config to be able to disable this whole init process

Reply:
Indeed, I forgot to check in the bus event handler if the initiator_thread is not None.
I worked on this issue a bit by mistake, I thought I had it in my sprint. That's why I converted it to a draft and did not go further with testing